src/worker.py

```
@@ -1301,4 +1337,155 @@ async def on_fetch(request, env):
+        " WHERE pc.addressee_id=? AND pc.status='pending'"
+        " ORDER BY pc.created_at DESC"
+    ).bind(user["id"]).all()
+    requests = []
+    for r in rows.results or []:
+        requests.append({
+            "connection_id": r["id"],
+            "requester_id": r["requester_id"],
+            "requester_name": await decrypt_aes(r["requester_name_enc"] or "", enc),
+            "message": r["message"] or "",
+            "requested_at": r["created_at"],
+        })
```

Ruff (0.15.7)

[warning] Use a list comprehension to create a transformed list

(PERF401)